docs/content/getting-started/overview.mdx

```diff
@@ -0,0 +1,68 @@
+---
+title: "Overview | Dagster Docs"
+description: ""
+---
+
+# Welcome to Dagster!
+
+If you’re here, you probably have questions about data, orchestration, Dagster, and how everything fits together.
+
+Let’s start with the basics: **What’s an orchestrator?**
+
+Orchestrators allow you to automatically execute a series of steps and track the results. They not only enable you and your team to see how long steps take to run or how they connect to other steps, but to understand when, where, and why things break.
+
+---
+
+## Why do I need a data orchestrator?
+
+What if you’re working with data? Unlike generic orchestrators that may need to accommodate a variety of use cases, data orchestrators focus on building and managing data pipelines.
+
+Data orchestrators allow you to automatically run, in the correct order, the steps in data pipelines that move, process, and transform data. A data orchestrator helps to ensure that your data assets - such as database tables, reports, language learning models, and so on - are accurate and up-to-date.
+
+That being said, you might be asking yourself...
+
+---
+
+## Why Dagster?
+
+Dagster models pipelines in terms of the data assets they produce and consume, which, by default, brings order and observability to your data platform.
+
+With Dagster, you declare functions in Python that you want to run and the data assets that those functions produce or update. For example, the following image is an example of a graph with several assets:
+
+<!-- ![Example of an asset graph](/images/getting-started/dagster-u-lesson-9-graph.png) -->
```

Comment:
I've observed that having the "kind" tags on assets (dbt, Spark, Pandas, etc.) makes a big difference in helping it click for people that Dagster brings all these technologies together.  Not a blocker to merging this, but wanted to advocate for this being worthwhile at least as a followup.

Reply:
I like it!